README.md

```diff
@@ -12,9 +12,9 @@ Simple Core API: [play_routes](docs/stardoc/play-routes.md)
 For more information about the Play Framework, see [the Play documentation](https://www.playframework.com/documentation/latest).
 
 ## Installation
-Create a file called at the top of your repository named `WORKSPACE` and add the following snippet to it.
+Create a file at the top of your repository named `WORKSPACE` and add the following snippet to it.
 
-```python
```

Comment:
I like the python highlight, why removing it :(

Reply:
I'll bring it back :)